### PR TITLE
Introduction To PHP Graphic Techniques 

### LINKED ISSUE #4768: Introduction to PHP graphic techniques
####Proposed title of article
[Languages] Introduction to PHP graphic techniques 

####Proposed title introduction
PHP has the Inherent ability to create and manipulate graphics independently. The scope and capabilities of PHP graphics handling is indeed deep and powerful. It does this with the aid of the GD library which currently supports image formats in JPEG, png, gif and wbmp. 
    This tutorial aims at giving a solid foundation to intended readers on the first steps to take in this graphics journey. It is an intended intro to follow up graphics tutorials that I intend to publish here. 

Key takeaways
i.	Readers will learn how to create a canvas, ink and paint brush.
ii.	Readers will learn how to draw geometric shapes such as rectangles, polygons e.t.c
iii.	Readers will learn how to write words on the canvas.

### PR DESCRIPTION
**Pull request by SarahAsuquo**
### Article checklist

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

`This closes #4768 `

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by SarahAsuquo** [comment]
Hello @WanjaMIKE,@jumamiller this is a duplicate of #4953. I have effected all the necessary changes and hopefully it is ready for review. 
Cc.
@hectorkambow

**Comment by jumamiller** [comment]
> Hello @WanjaMIKE,@jumamiller this is a duplicate of #4953. I have effected all the necessary changes and hopefully it is ready for review. Cc. @hectorkambow

Seems you got multiple files?

**Comment by SarahAsuquo** [comment]
@jumamiller sorry, I don't quite understand your question. My intention is to resubmit an already closed pull request that needed changes. Could you please delete the multiple files for me if they're not needed.

**Comment by hectorkambow** [comment]
@SarahAsuquo  

Please look at our [resources ](https://github.com/section-engineering-education/engineering-education/tree/master/new_contributors)to see how to go about fixing this 👍 
 Thank you

**Comment by hectorkambow** [comment]
The pull request should have only the file(s) [index file and any corresponding images] you plan on submitting and having published. 🚀

**Comment by hectorkambow** [comment]
Not sure which one needs to be deleted (are the merely duplicates) copies??

File: content/articles/introduction-to-php-graphic-techniques/index.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
+ Introduction To PHP Graphic Techniques
+### Introduction
+PHP has the inherent ability to create and manipulate graphics independently. The scope and capabilities of PHP graphic handling is quite deep and powerful. It does this with the aid of the GD library which currently supports image formats in jpeg, png, gif and wbmp. 
+This tutorial aims at providing a solid foundation on the first steps to take to start doing exploits with PHP graphics. If you’re as excited as I am, then let’s proceed. 
+
+### Prerequisites
+To follow along with this tutorial, you need to have the following:
+•	 A code editor like Visual Studio Code[link].
```
Comment (jumamiller) [inline content/articles/introduction-to-php-graphic-techniques/index.md:8]: The link seems to be incomplete?

File: content/articles/introduction-to-php-graphic-techniques/index.md [hunk 2]
```diff
@@ -0,0 +1,223 @@
+ Introduction To PHP Graphic Techniques
+### Introduction
+PHP has the inherent ability to create and manipulate graphics independently. The scope and capabilities of PHP graphic handling is quite deep and powerful. It does this with the aid of the GD library which currently supports image formats in jpeg, png, gif and wbmp. 
+This tutorial aims at providing a solid foundation on the first steps to take to start doing exploits with PHP graphics. If you’re as excited as I am, then let’s proceed. 
+
+### Prerequisites
+To follow along with this tutorial, you need to have the following:
+•	 A code editor like Visual Studio Code[link].
+•	 Apache web server, Preferably WAMP[link] or XAMPP[link].
```
Comment (jumamiller) [inline content/articles/introduction-to-php-graphic-techniques/index.md:9]: XAMPP/WAMPP is a stack, are they really servers?

**Review by jumamiller — CHANGES_REQUESTED** [review]
Kindly explain all code snippets as much as possible.

**Comment by SarahAsuquo** [comment]
@jumamiller I have made the necessary corrections.

File: content/articles/introduction-to-php-graphic-techniques/index.md (hunk 2 above)
Comment (SarahAsuquo) [reply content/articles/introduction-to-php-graphic-techniques/index.md]: I'm sorry for the mistake, it was a typographical error. I've corrected it in the updated file.

**Comment by hectorkambow** [comment]
@SarahAsuquo our peer reviewer will jump on the updates asap - he is busy reviewing other pull request as well. 

Thank you for your patience as our team is working diligently to work through the increased volume of articles.

**Comment by hectorkambow** [comment]
@jumamiller 
can we get a status on this PR ? 

If any more major revisions are needed - we will be closing this PR asap. 👀

File: content/articles/introduction-to-php-graphic-techniques/index.md [hunk 3]
```diff
@@ -0,0 +1,227 @@
+ Introduction To PHP Graphic Techniques
+### Introduction
+PHP has the inherent ability to create and manipulate graphics independently. The scope and capabilities of PHP graphic handling is quite deep and powerful. It does this with the aid of the GD library which currently supports image formats in jpeg, png, gif and wbmp.
```
Comment (jumamiller) [inline content/articles/introduction-to-php-graphic-techniques/index.md:3]: - This should be a standalone paragraph. Also, The second sentence should have some transitions to aid in article flow. For example, `....Therefore, the scope and capabilities`,.

-  Fix singular verb that doesn't agree with plural compound subject.
- Make good use of commas.
-  Quote unrecognised English words.

File: content/articles/introduction-to-php-graphic-techniques/index.md [hunk 4]
```diff
@@ -0,0 +1,227 @@
+ Introduction To PHP Graphic Techniques
+### Introduction
+PHP has the inherent ability to create and manipulate graphics independently. The scope and capabilities of PHP graphic handling is quite deep and powerful. It does this with the aid of the GD library which currently supports image formats in jpeg, png, gif and wbmp. 
+This tutorial aims at providing a solid foundation on the first steps to take to start doing exploits with PHP graphics. If you’re as excited as I am, then let’s proceed.
```
Comment (jumamiller) [inline content/articles/introduction-to-php-graphic-techniques/index.md:4]: This sentence here is not quite clear and it's hard to follow. Let's have a simple sentence.

For example `This tutorial aims to provide a solid foundation on the first steps to start doing exploits with PHP graphics.` is much better that what you currently have!

File: content/articles/introduction-to-php-graphic-techniques/index.md [hunk 5]
```diff
@@ -0,0 +1,227 @@
+ Introduction To PHP Graphic Techniques
+### Introduction
+PHP has the inherent ability to create and manipulate graphics independently. The scope and capabilities of PHP graphic handling is quite deep and powerful. It does this with the aid of the GD library which currently supports image formats in jpeg, png, gif and wbmp. 
+This tutorial aims at providing a solid foundation on the first steps to take to start doing exploits with PHP graphics. If you’re as excited as I am, then let’s proceed. 
+
+### Prerequisites
+To follow along with this tutorial, you need to have the following:
+•	 A code editor like [Visual Studio Code](https://code.visualstudio.com/).
+•	 Apache web server, Preferably [WampServer](https://www.wampserver.com/en/).
```
Comment (jumamiller) [inline content/articles/introduction-to-php-graphic-techniques/index.md:9]: Ensure you write mnemonics correctly as they appear in documentations!

File: content/articles/introduction-to-php-graphic-techniques/index.md [hunk 6]
```diff
@@ -0,0 +1,227 @@
+ Introduction To PHP Graphic Techniques
+### Introduction
+PHP has the inherent ability to create and manipulate graphics independently. The scope and capabilities of PHP graphic handling is quite deep and powerful. It does this with the aid of the GD library which currently supports image formats in jpeg, png, gif and wbmp. 
+This tutorial aims at providing a solid foundation on the first steps to take to start doing exploits with PHP graphics. If you’re as excited as I am, then let’s proceed. 
+
+### Prerequisites
+To follow along with this tutorial, you need to have the following:
+•	 A code editor like [Visual Studio Code](https://code.visualstudio.com/).
+•	 Apache web server, Preferably [WampServer](https://www.wampserver.com/en/).
+•	 GD library/extension. You can confirm if you have it installed by searching for “gd” in your php.ini file. You should find “etension=php_gd2.dll” else download it for free on the internet.
```
Comment (jumamiller) [inline content/articles/introduction-to-php-graphic-techniques/index.md:10]: What does `etension` in PHP?

File: content/articles/introduction-to-php-graphic-techniques/index.md [hunk 7]
```diff
@@ -0,0 +1,227 @@
...
+This tutorial aims at providing a solid foundation on the first steps to take to start doing exploits with PHP graphics. If you’re as excited as I am, then let’s proceed. 
+
+### Prerequisites
+To follow along with this tutorial, you need to have the following:
+•	 A code editor like [Visual Studio Code](https://code.visualstudio.com/).
+•	 Apache web server, Preferably [WampServer](https://www.wampserver.com/en/).
+•	 GD library/extension. You can confirm if you have it installed by searching for “gd” in your php.ini file. You should find “etension=php_gd2.dll” else download it for free on the internet. 
+•	Basic knowledge of PHP.
+### Create a Canvas, Ink and Paint Brush
+Assuming you are a conventional artist and you want to draw a picture, you will likely require three things: a canvas to draw on, a paint brush to draw with and you’ll need ink for the paint brush to use. That is the same convention we employ while working with PHP graphics and thankfully, the GD extension offers us functions that can serve as the canvas, paint brush and ink.
```
Comment (jumamiller) [inline content/articles/introduction-to-php-graphic-techniques/index.md:13]: Punctuations!

**Review by jumamiller — CHANGES_REQUESTED** [review]
Hi, kindly rework on this article to fix punctuation marks mistakes. Reorganise your article to have flow, this includes avoiding wordy words and breaking them into distinct paragraphs!

Meanwhile the changes are quite a number, that I feel you should revise the entire work to fix your grammar as well. This helps in fixing minor singular ,plural mistakes et cetera.

Feel free to reach me on Slack (@Miller).

**Comment by jumamiller** [comment]
> Hi, kindly rework on this article to fix punctuation marks mistakes. Reorganise your article to have flow, this includes avoiding wordy words and breaking them into distinct paragraphs!
> 
> Meanwhile the changes are quite a number, that I feel you should revise the entire work to fix your grammar as well. This helps in fixing minor singular ,plural mistakes et cetera.
> 
> Feel free to reach me on Slack (@miller).

still waiting for updates!

**Comment by SarahAsuquo** [comment]
@jumamiller please I'm working on the corrections currently, I'll be done in a few hours. Thanks for your patience.

**Comment by SarahAsuquo** [comment]
@jumamiller i have made the requested updates to the file. Please take a look.

File: content/articles/introduction-to-php-graphic-techniques/index.md [hunk 8]
```diff
@@ -0,0 +1,220 @@
+ Introduction To PHP Graphic Techniques
+### Introduction
+PHP has the inherent ability to create and manipulate graphics independently. Therefore, the scope and capabilities of PHP graphic handling are quite deep and powerful. It does this with the aid of the GD library, which currently supports image formats in ‘jpeg’, ‘png’, ‘gif’ and ‘wbmp’. 
+This tutorial aims to provide a solid foundation on the first steps to take to start doing exploits with PHP graphics. If you’re as excited as I am, then let’s proceed. 
+### Prerequisites
+To follow along with this tutorial, you need to have the following:
+•	A code editor like [Visual Studio Code](https://code.visualstudio.com/).
+•	Apache web server, preferably [WAMPServer](https://www.wampserver.com/en/).
+•	GD library/extension. You can confirm if you have it installed by searching for “gd” in your php.ini file. You should find “extension=php_gd2.dll” else download it for free on the internet.
```
Comment (jumamiller) [inline content/articles/introduction-to-php-graphic-techniques/index.md:9]: Fix formatting issues here..

**Review by jumamiller — CHANGES_REQUESTED** [review]
Hello @SarahAsuquo which updates have you made? I can see you have very very long paragraphs...kindly break them down. Revise the whole article, just take your time and polish it well! If you have any query, contact me on Slack.

FYI: Any further revision from this step will result in article closure!

**Comment by SarahAsuquo** [comment]
@jumamiller I have given the article a total review and changes have been made. I sincerely hope it meets standard this time.

**Comment by jumamiller** [comment]
Hi, reach me on slack @Miller.

**Comment by hectorkambow** [comment]
@SarahAsuquo 
Good afternoon and thank you for submitting your topic suggestion. Not a problem but it seems that this PR is still quite a bit aways from being completed. 
 
In an effort to be fair to our larger EngEd community as a rule of thumb - we can only accept content (pull requests) that are complete and fully polished articles/tutorial - meaning they are ready to be published. 
 
All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) - this allow all revisions to be done on your forked personal repo. 
 
We will be closing this PR and please re-submit it when it is ready to be published as a new PR (and you can add the duplicate label) and ping the reviewer and myself. 
 
>NOTE: A duplicate PR will go back into the queue and will not receive any further major revisions. A duplicate PR should be 100% ready to be published, otherwise it will be closed immediately. We encourage our new contributors to go over our [resources page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) and [new contributor folder](https://github.com/section-engineering-education/engineering-education/tree/master/new_contributors) to be better familiar with the level of quality and style we are looking to publish.
 
Closing a PR does NOT mean the article can not be published (or fixed) it simply means that the edits, revisions, and fixes will happen outside of the queue.
 
👍 
cc @jumamiller

**Pull request closed without merging**